Gopkg.toml

```diff
@@ -60,3 +60,7 @@ required = [
 [[constraint]]
   name = "github.com/operator-framework/operator-lifecycle-manager"
   version = "0.7.1"
+
+[[constraint]]  
+  branch = "master"
+  name = "github.com/operator-framework/go-appr"
```

Comment:
If I remember correctly, I did 
```
dep ensure -add github.com/operator-framework/go-appr
```

Reply:
Reason for these questions was to figure out why `Gopkg.lock` changed drastically and we now know it is was because I was using an older version of `dep`.